jwst/datamodels/drizproduct.py

```diff
@@ -1,30 +0,0 @@
-from .model_base import DataModel
```

Comment:
Need to also remove `DrizProductModel` and `MultiProductModel` from `datamodels/__init__.py`, as well as any occurrences in docs.